htdocs/admin/stock.php

```diff
@@ -357,7 +365,7 @@
 print '<td class="right">';
 if (isModEnabled("supplier_order") || isModEnabled("supplier_invoice")) {
 	if ($conf->use_javascript_ajax) {
-		if ($disabled) {
+		if (in_array('VALIDATE_ORDER', $disableStockCalculateOn)) {
```

Comment:
I think you want to do here ?
if (in_array('SUPPLIER_VALIDATE_ORDER', $disableStockCalculateOn)) {

Reply:
Yes, you're right. 
I have done it.